src/renuka/Shop.java

```diff
@@ -0,0 +1,98 @@
+package renuka;
+
+public class Shop {
+		
+	 int maggieQ;
+	 int dosaQ;
+	 int pouchesQ;
+	 int panipuriQ;
+	 int masalaQ;
+	
+	
+	void setQ(int reqMaggieQ, int reqDosaQ, int reqPouchesQ, int reqPanipuriQ, int reqMasalaQ) {
+		maggieQ = reqMaggieQ;
+		dosaQ = reqDosaQ;
+		pouchesQ = reqPouchesQ;
+		panipuriQ = reqPanipuriQ;
+		masalaQ = reqMasalaQ;
+	}	
+	
+	void requestedQ(int reqMaggieQ, int reqDosaQ, int reqPouchesQ, int reqPanipuriQ, int reqMasalaQ) {
+		if(reqMaggieQ <= maggieQ) {
+		maggieQ = maggieQ - reqMaggieQ;
+		System.out.println("Remaining Qty is: " +maggieQ);
+		}
+		else {
```

Comment:
We should not print that maggie is not available instead of this we should print we should print available stock.